incubator/cassandra/values.yaml

```diff
@@ -0,0 +1,58 @@
+## Cassandra image version
+## ref: https://hub.docker.com/r/library/cassandra/
+image:
+  repo: "cassandra"
+  tag: "latest"
```

Comment:
I would like to use `latest` as default value so we do not need to update chart every time when new version is released. User can still specify the version they need.